Sources/DistributedCluster/ClusterSystemSettings.swift

```diff
@@ -404,50 +404,96 @@ protocol ClusterSystemInstrumentationProvider {
+            }
+        )
+    }
+
+    /// Locate the default `ClusterD` process and use it for discovering cluster nodes.
+    public static var clusterd: Self {
+        get {
+            Self.clusterd(endpoint: nil)
         }
     }
 
+    public static func clusterd(endpoint: Cluster.Endpoint?) -> Self {
+        ServiceDiscoverySettings(clusterdEndpoint: endpoint ?? ClusterDaemon.defaultEndpoint)
+    }
+
+    public static func seed(nodes: Set<Cluster.Endpoint>) -> Self {
+        .init(static: nodes)
+    }
+
     public init(static nodes: Set<Cluster.Endpoint>) {
-        self.implementation = .static(nodes)
-        self._subscribe = { onNext, _ in
-            // Call onNext once and never again since the list of nodes doesn't change
-            onNext(.success(Array(nodes)))
-            // Ignore onComplete because static service discovery never terminates
-
-            // No cancellation token
-            return nil
-        }
+        self.implementation = .static(
+            endpoints: nodes,
+            subscribe: { onNext, _ in
+                // Call onNext once and never again since the list of nodes doesn't change
+                onNext(.success(Array(nodes)))
+                // Ignore onComplete because static service discovery never terminates
+
+                // No cancellation token
+                return nil
+            }
+        )
     }
 
     /// Similar to `ServiceDiscovery.subscribe` however it allows the handling of the listings to be generic and handled by the cluster system.
     /// This function is only intended for internal use by the `DiscoveryShell`.
     func subscribe(onNext nextResultHandler: @escaping (Result<[Cluster.Endpoint], Error>) -> Void, onComplete completionHandler: @escaping (CompletionReason) -> Void) -> CancellationToken? {
-        self._subscribe(nextResultHandler, completionHandler)
+        switch self.implementation {
+        case .static(_, let subscribe),
+             .dynamic(_, let subscribe):
+            subscribe(nextResultHandler, completionHandler)
+        case .clusterDaemon:
+            .none
+        }
+    }
+
+    func initializeClusterd(_ system: ClusterSystem) {
+        switch self.implementation {
+        case .clusterDaemon(_, let initialize):
+            initialize(system)
+        default:
+            break
+        }
     }
 
     enum ServiceDiscoveryImplementation {
-        case `static`(Set<Cluster.Endpoint>)
-        case dynamic(AnyServiceDiscovery)
+        case `static`(endpoints: Set<Cluster.Endpoint>, subscribe: (@escaping (Result<[Cluster.Endpoint], Error>) -> Void, @escaping (CompletionReason) -> Void) -> CancellationToken?)
+        case dynamic(serviceDiscovery: AnyServiceDiscovery, subscribe: (@escaping (Result<[Cluster.Endpoint], Error>) -> Void, @escaping (CompletionReason) -> Void) -> CancellationToken?)
+        case clusterDaemon(endpoint: Cluster.Endpoint, initialize: (ClusterSystem) -> Void)
```

Comment:
Decided to put subscribe and initialize closures here, as otherwise it's quite confusing to have optional values in settings itself.